homeworks/serhii.noskov_SerhiiNSN/GamesProject/css/style.css

```diff
@@ -172,3 +172,313 @@ a {
+  display: inline-flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.custom-radio + label::before {
+  content: "";
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  border: 1px solid #FFF;
+  border-radius: 50%;
+  margin-right: 12px;
+  background-color: #000;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 50% 50%;
+}
+
+.check ~ label::before {
+  content: "\2713";
+  text-align: center;
+  color: #FFF;
+  line-height: 1;
+  width: 16px;
+  height: 16px;
+  border: 2px solid #EF4934;
+  border-radius: 6px;
+  margin-right: 12px;
+  display: inline-block;
+}
+
+.custom-radio:checked + label::before {
+  border-color: #FFF;
+  background-color: #000;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3e%3ccircle r='3' fill='%23fff'/%3e%3c/svg%3e");
+}
+
+.checkbox-container {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  color: #FFF;
+  cursor: pointer;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.chekbox-block {
+  display: flex;
+  align-items: center;
+  color: #FFF;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  column-gap: 32px;
+}
+
+.checkbox-container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark {
+  position: absolute;
+  top: -2px;
+  left: 0;
+  height: 20px;
+  width: 20px;
+  background-color: #000;
+  border: 1px solid #FFF;
+  border-radius: 6px;
+}
+
+.checkmark::after {
+  content: "";
+  position: absolute;
+  display: none;
+}
+
+.checkbox-container input:checked ~ .checkmark::after {
+  display: block;
+}
+
+.checkbox-container input:checked ~ .checkmark::after {
+  left: 7px;
+  top: 2px;
+  width: 5px;
+  height: 10px;
+  border: solid #FFF;
+  border-width: 0 3px 3px 0;
+  transform: rotate(45deg);
+}
+
+.search-area {
+  position: relative;
+  display: flex;
+}
+
+.search {
+  padding: 14px 14px 14px 46px;
+  outline: none;
+  border: none;
+  border-radius: 6px 0 0 6px;
+  background: #FFF;
+  width: 240px;
+}
+
+.search-img {
+  position: absolute;
+  top: 50%;
+  transform: translate(50%, -50%);
+}
+
+.submit {
+  padding: 13px 20px;
+  border-radius: 0 6px 6px 0;
+  background: #EF4934;
+  border: none;
+  color: #FFF;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  cursor: pointer;
+}
+
+/* ========================== LOGIN ============================= */
+
+.login__container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 50px;
+}
+
+.login__tittle {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 48px;
+  font-weight: 700;
+  margin: 80px 0 60px;
+  text-align: center;
+}
+
+.email__text,
+.email__password {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 22px;
+  font-weight: 600;
+  margin-bottom: 22px;
+}
+
+.email__input,
+.password__input {
+  padding: 30px 380px 30px 20px;
```

Comment:
I think you should make `padding-right` equal to `padding-left` here. Otherwise the long email would be cutted.
<img width="1471" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/c599b984-16e6-4485-bae9-2953defcd030">